.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,72 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-9.0",
+	"features": {
+		"ghcr.io/devcontainers/features/azure-cli:1": {},
+		"ghcr.io/azure/azure-dev/azd:0": {},
+		"ghcr.io/devcontainers/features/docker-in-docker": {},
+		"ghcr.io/devcontainers/features/dotnet": {
+			"additionalVersions": [
+				"8.0.403"
+			]
+		}
+	},
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		15887, // playground/waitfor AppHost
+		5180, // playground/waitfor ApiService
+		7024, // playground/waitfor Frontend
+		15551 // playground/waitfor PGAdmin
+	],
+	"portsAttributes": {
+		"15887": {
+			"label": "WaitFor Playground: AppHost",
+			"protocol": "https"
+		},
+		"5180": {
+			"label": "WaitFor Playground: ApiService",
+			"protocol": "http"
+		},
+		"7024": {
+			"label": "WaitFor Playground: Frontend",
+			"protocol": "https"
+		},
+		"15551": {
+			"label": "WaitFor Playground: PGAdmin",
+			"protocol": "http"
+		}
+	},
+	"otherPortsAttributes": {
+		"onAutoForward": "ignore"
```

Comment:
This is supposed to stop autoforwarding although it doesn't seem to be reliable.